### ISSUE TITLE
Support tags on aws_spot_fleet_request

### ISSUE THREAD

**Issue by emman27** [terraform-providers/terraform-provider-aws#6627]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

As mentioned. As of now, tags can only be attached to the EC2 instances spun up but not the request itself.

Also see https://github.com/terraform-providers/terraform-provider-aws/pull/2042#issuecomment-402967969

### New or Affected Resource(s)

* aws_spot_fleet_request

### Potential Terraform Configuration

```hcl
resource "aws_spot_fleet_request" "req" {
  #... other configurations
  tags = {
        Name = "${var.name}"
    }
}
```

### References

* #0000

**Comment by jkburges** [comment]
Is it even possible to tag a spot fleet request itself in AWS? 

I can't see anything about it in any doc or APIs (so I think it may be a limitation of AWS rather than this terraform provider).

**Comment by electroly** [comment]
It is now: https://aws.amazon.com/about-aws/whats-new/2020/02/amazon-ec2-now-supports-tagging-ec2-spot-fleet-requests/

**Comment by DrFaust92** [comment]
addressed in #12295

**Comment by bflad** [comment]
Support for tagging Spot Fleet Requests has been merged and will release with version 2.57.0 of the Terraform AWS Provider, later today. Thanks to @DrFaust92 for the implementation. 👍
